app/tests/e2e/test_connections.py

```diff
@@ -39,6 +42,16 @@ async def test_accept_invitation(
     )
     connection_record = accept_response.json()
 
+    assert check_webhook_state(
+        client=alice_member_client,
+        desired_state={"state": "completed"},
+        topic="connections",
+        filter_map=FilterMap(
+            filter_key="connection_id",
+            filter_value=connection_record["connection_id"],
+        ),
```

Comment:
sure - good idea!

Reply:
Also remember now whilst changing this why i did this. It's because when I use this to filter I can use filter_map.filter_key or filter_map.filter_value and be sure it works. WHere as if I have a dict and you can't be sure because you there can be multiple keys/ key value pairs etc

Reply:
another one: I kept desired_state separate because then the check can also be "sth" (key) instead of "state" key

Reply:
I meant that if we dont want to check for "state": "completed" but "some_other_key": "some_other_value" that was imho more obvious/readable. Changed it to your proposal now though.